fl4health/clients/nnunet_client.py

```diff
@@ -141,6 +143,10 @@ def __init__(
                 provided. Defaults to None.
             reporters (Sequence[BaseReporter], optional): A sequence of FL4Health
                 reporters which the client should send data to.
+            nnunet_trainer_class (Type[nnUNetTrainer]): A nnUNetTrainer constructor.
```

Comment:
Maybe add a quick note here and on the server-side that the training classes have to match each other (client and server trainers should be the same)?